CHANGELOG.md

```diff
@@ -6,6 +6,128 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0] - June, 27, 2022
+
+### Design Tokens
+#### Breaking Changes
```

Comment:
The breaking changes heading is level 4 heading but there is nothing under it. The next heading is also level 4. So are the below changes breaking changes or are there breaking changes at all?

Reply:
I copied this from v1.0.0 format